### PR TITLE
DS-4122 - Remove libraries and refactor search script

### PR DESCRIPTION
**Pull request by MaikelGG**
- Updated waves script, so it can be easily updated
- Removed modified version of contrib search script
- Add new interaction for search block in navbar
- The block is in the conten top region, and hidden. There is a trigger to show the form
- On mobile the form remains the same
- Remove contrib scripts that are not used
- Added a gulp watch task for scripts

### CODE REVIEW

**Comment by MaikelGG** [comment]
@bramtenhove Travis fails because of missing composer dependencies of drupal_social feature branch

File: themes/socialbase/src/Plugin/Preprocess/SocialBaseBlock.php [hunk 1]
```diff
@@ -128,11 +128,15 @@ public function preprocess(array &$variables, $hook, array $info) {
...
-      $variables['content']['search_form']['actions']['submit']['#addsearchicon'] = TRUE;
-
       if ($region == 'hero') {
         $variables['content']['search_form']['#attributes']['class'][] = 'hero-form';
         $variables['content']['search_form']['#region'] = 'hero';
+        $variables['content']['search_form']['actions']['submit']['#addsearchicon'] = TRUE;
+      }
+      elseif ($region == 'content_top') {
+        $variables['content']['search_form']['#region'] = 'content-top';
+        $variables['content']['search_form']['search_input_content']['#attributes']['placeholder'] = t('What are looking for ?');
```
Comment (bramtenhove) [inline themes/socialbase/src/Plugin/Preprocess/SocialBaseBlock.php]: @MaikelGG, the text is incorrect. Could you change it to something like `What are you looking for?`

**Comment by bramtenhove** [comment]
@MaikelGG, as the interaction changes for the search form I'd like another pair of eyes. Who do you suggest can do this best?

**Comment by MaikelGG** [comment]
@bramtenhove I can demo it to Xinyu next week. Maybe Moritz can join too.

**Comment by MaikelGG** [comment]
I have updated the text

**Comment by bramtenhove** [comment]
Alright, sounds like a plan. I'll complete testing the functionality and will then run all the stability tests locally.

**Pull request merged**